SECURITY.WHITEPAPER.md

```diff
@@ -0,0 +1,113 @@
+# Datasafe Security Whitepaper
+
+## General information
+Datasafe is a flexible encryption library. It uses different encryption algorithms. Some of them can be 
+configured by client application. It uses encryption algorithms provided by Bouncy Castle library by default.
```

Comment:
`It uses encryption algorithms provided by Bouncy Castle library by default`
Under the hood Datasafe uses BouncyCastle library to perform encryption.